src/main/resources/templates/product/productOverview.html

```diff
@@ -186,8 +186,46 @@
                             </form>
                         </div>
                     </div>
+                    <!--category-filter-->
+                    <select onchange="selectCategory(this)" class="category-dropdown category-select"
+                            id="categorySelect" name="categorySelect">
+                        <option disabled selected value="none" style="color: black">Search for a category here...
+                        </option>
+                        <option th:each="category : ${searchModel.getCategories()}" style="color: black"
+                                th:value="${category}"
+                                th:text="${category.name}">
+                        </option>
+                    </select>
+                    <script th:inline="javascript">
+                        function selectCategory(selectedCategory) {
+                            const category = selectedCategory.options[selectedCategory.selectedIndex].text;
+                            location = "/search/category/?category=" + category;
+                        }
+                    </script>
+                    <div class="selected-categories">
+                        <div th:if="${searchModel.selectedCategories.size() > 0}"
```

Comment:
please hide this if no categories are selected, there's a horizontal scollbar visible now even if no categories are selected

Reply:
oops... if-check stood one <div> too low. Fixed